src/tests/App.test.js

```diff
@@ -12,4 +12,34 @@ describe('<App />', () => {
     const div = document.createElement('div');
     ReactDOM.render(<App />, div);
   });
+
+  it('should have two children', () => {
+    const app = shallow(<App />);
+    expect(app.find('div').children()).toHaveLength(2);
+  });
+
+  it('should render the tag Display', () => {
+    const app = shallow(<App />);
+    expect(app.find('Display')).toHaveLength(1);
```

Comment:
Looks good.
Consider this though 
``` const app = shallow(<App />);
    let children = app.find('div').children();
    expect(children).toHaveLength(2);```
We try to break down the problem to smaller pieces.